docs/policy/balloons.md

```diff
@@ -85,6 +85,13 @@ Balloons policy parameters:
   pack new balloons tightly into the same NUMAs/dies/packages. This
   helps keeping large portions of hardware idle and entering into deep
   power saving states.
+- `PreferMoreCores` prefers allocating logical CPUs for a balloon from
```

Comment:
Maybe somehow name to reference physical vs hyperthread "cores"?

Reply:
`PreferSpreadOnPhysicalCores`